src/controllers/signAccountOp/signAccountOp.ts

```diff
@@ -492,14 +486,23 @@ export class SignAccountOpController extends EventEmitter implements ISignAccoun
   }
 
   async #reestimate() {
+    // stop the interval reestimate if the user has done it over 20 times
+    this.#reestimateCounter += 1
+    if (this.#reestimateCounter > 20) this.#stopRefetching = true
+
```

Comment:
the counter is not being reset when the calls change. This results in stopRefetching remaining `true` even after the user modifies the swap/transfer form

Reply:
adding it to "update()" when the calls have changed